string type		# message type {NewMission, Restart, Abort, Completed}
string ag_addr  	# agent address (namespace) - sender
int32 mission_id
string mission_type	# mission type {Homogeneous, Heterogeneous} - if Homogeneous -> mission schedule can't be interrupted with other mission's tasks
string root_task
string criteria
